manage.py

```diff
@@ -66,13 +80,30 @@ def _make_context():
         'User': User
     }
 
-@manager.command
-def test():
-    """Run the tests."""
+def server_test():
+    """Run the server side tests"""
     import pytest
     exit_code = pytest.main(['tests', '--verbose'])
     return exit_code
 
+def client_test():
+    """Run the frontend tests."""
+    import subprocess
+    return subprocess.call(["node_modules/karma/bin/karma", "start"])
+
+@manager.option('-t', '--type', dest='type', default='all')
+def test(type):
+    """Run the tests."""
+    print type
+    if type == "server":
+        return server_test()
+    elif type == "client":
+        return client_test()
+    else:
+        server_exit_code = server_test()
+        client_exit_code = client_test()
+        return server_exit_code and client_exit_code
```

Comment:
Assuming this returns 0 on success (haven't tried it), should this be `server_exit_code or client_exit_code` to give the worst of the two?